config.json

```diff
@@ -47,6 +47,16 @@
       ]
     },
     {
+      "uuid": "65a76aba-a485-222d-84ba-e9a445b25be6",
+      "slug": "collatz-conjecture",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 1,
```

Comment:
Maybe change the difficulty to 2? 1 seems a tad low.
And, maybe change unlocked_by to "leap"